backend/.env.example

```
@@ -27,6 +27,12 @@ AUCTION_CONTRACT_ID=C... # Auction mechanism contract address
 # --- API Authentication ---
 API_KEYS=key1,key2,key3 # Comma-separated list of valid API keys for guarding routes
 
+TYPEORM_SYNCHRONIZE=true
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The TYPEORM_SYNCHRONIZE key is duplicated

(DuplicatedKey)